app/components/app/Header/Header.css

```diff
@@ -116,10 +116,25 @@ header {
     background: url(campaign.jpg) no-repeat;
     background-size: cover;
     line-height: 50px;
-    font-size: 20px;
+    font-size: 1em;
     padding: 0 50px;
     text-align: center;
+    z-index: 50;
 }
 .campaign-button {
     margin-left: 50px;
+    margin-right: 10px;
+}
+.campaign-close {
+    cursor: pointer;
+    cursor: hand;
```

Comment:
Cursor `pointer` or cursor `hand`? It's not necessary to use both anymore: http://stackoverflow.com/a/3088012/5708679